core/client/fs/src/main/java/alluxio/util/FileSystemOptions.java

```diff
@@ -42,6 +42,17 @@
  * will populate the gRPC options objects with the proper values based on the given configuration.
  */
 public class FileSystemOptions {
+  /**
+   * @param conf Alluxio configuration
+   * @return options based on the configuration
+   */
+  public static ScheduleAsyncPersistencePOptions scheduleAsyncPersistenceDefaults(
+      AlluxioConfiguration conf) {
+    return ScheduleAsyncPersistencePOptions.newBuilder()
+        .setCommonOptions(commonDefaults(conf))
+        .setPersistenceWaitTime(0)
```

Comment:
Do we always want this to be `0`? Shouldn't it be set based on the configuration?

Reply:
If a user directly call persist, they usually want the file/directory to persist immediately. If not, they can just set the wait time through CLI.